service-war/src/main/java/com/redshiftsoft/tesla/web/filter/CookieHelper.java

```diff
@@ -26,16 +29,16 @@ public static Cookie getCookie(final HttpServletRequest request, String cookieNa
     }
 
     /**
-     * Add the specified cookie to the the response.
+     * Add the specified cookie builder to the the response.
      */
-    public static void addCookie(final HttpServletRequest request, final HttpServletResponse response, Cookie cookie) {
+    public static void addCookie(final HttpServletRequest request, final HttpServletResponse response, ResponseCookie.ResponseCookieBuilder cookie) {
         /* Important security measure: see design decisions on cookie security. */
         if (request.isSecure()) {
-            cookie.setSecure(true);
+            cookie = cookie.secure(true);
         } else if ("development".equals(System.getProperty("spring.profiles.active"))) {
-            cookie.setSecure(false);
+            cookie = cookie.secure(false);
         }
-        response.addCookie(cookie);
+        response.addHeader(HttpHeaders.SET_COOKIE, cookie.sameSite("Lax").build().toString());
```

Comment:
We should probably add a comment explaining why we're using `Lax` (and not `Strict`)